libs/vkd3d/meta_commands.c

```diff
@@ -400,6 +405,8 @@ static void d3d12_meta_command_exec_dstorage(struct d3d12_meta_command *meta_com
             parameters->status_buffer_va, parameters->status_buffer_size,
             parameters->stream_count);
 
+    d3d12_command_list_debug_mark_begin_region(list, "DStorage");
```

Comment:
I think we're missing a root parameter invalidate here?

Reply:
`ExecuteMetaCommands` calls `d3d12_command_list_invalidate_all_state`, that should be enough?

Reply:
Ah, right ...